content/en/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Monitoring Control Plane Components
+reviewers:
+- brancz
+- logicalhan
+- RainbowMango
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+System component metrics can give a better look into what is happening inside them. Metrics are particularly useful for building dashboards and alerts.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Metrics in Kubernetes
+
+Kubernetes control plane components use prometheus as the default client for exposing metrics. In most cases those metrics are available on `/metrics` endpoint of the HTTP server. For components that doesn't expose endpoint by default it can be enabled using `--bind-address` flag.
+
+Examples of those components:
+* kube-controller-manager
+* kube-proxy
+* kube-apiserver
+* kube-scheduler
+* kubelet (metrics exposed on `/metrics/cadvisor` and `/metrics/resource` do not have same guarantee)
+
+For clusters with RBAC enabled accessing metrics requires authorization through service account with `ClusterRole` for `/metrics` url.
+
+```
+apiVersion: rbac.authorization.k8s.io/v1	
+kind: ClusterRole	
+metadata:	
+  name: prometheus	
+rules:	
+  - nonResourceURLs:	
+      - "/metrics"	
+    verbs:	
+      - get
+```
+
+## Metric Lifecycle 
+
+Alpha metric -> Stable metric -> Deprecated metric -> Hidden metric -> Deletion
+
+Alpha metrics have no stability guarantees; as such they can be modified or deleted at any time.
+
+Stable metrics can be guaranteed to not change, except that the metric may become marked deprecated for a future Kubernetes version. By not change, we mean three things:
+
+* the metric itself will not be deleted (or renamed)
+* the type of metric will not be modified
+* no labels can be added or removed from this metric
+
+List of currently supported stable metrics can be found [here](https://github.com/kubernetes/kubernetes/blob/master/test/instrumentation/testdata/stable-metrics-list.yaml)
+
+Deprecated metric are annotated with a Kubernetes version, from which point that metric will be considered deprecated. When a stable metric undergoes the deprecation process, we are signaling that the metric will eventually be deleted. 
+
+Before deprecation:
+
+```
+# HELP some_counter this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+After deprecation:
+
+```
+# HELP some_counter (Deprecated from 1.15) this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Hidden metrics will no longer be exposed by default and will require manual component configuration by cluster administrators;
```

Comment:
```suggestion
Hidden metrics will no longer be exposed by default; to use a hidden metric, you need to override the configuration for the relevant cluster component.
```
(from [address the reader as “you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you))

Note that this changes the grammar; if you adopt this suggestion, other related lines will need to end with periods rather than semicolons.